src/lib/loader.js

```diff
@@ -142,46 +168,42 @@ export async function swapContent({firstRun, url, signal, ready, state}) {
   // Kick off loading the correct JS entrypoint.
   const entrypointPromise = loadEntrypoint(url);
 
-  // If this is the first run, bail out early. We generate an inferred partial for back/forward nav,
+  // If this is the first run, bail out early. We generate an inferred payload for back/forward nav,
```

Comment:
The term `payload` is kind of ambiguous, I'm wondering if there's a better name we could use.

Is it essentially the new page? Would newPage or something along those lines work?